### PR TITLE
add github workflow for running tests for jaseci studio

### PR DESCRIPTION
**Pull request by Gim3l**
Closes: #750

### CODE REVIEW

File: jaseci_studio/__e2e__/logs.spec.ts [hunk 1]
```diff
@@ -7,12 +7,12 @@ test("logs page loads", async ({ page }) => {
 
 test("logs messages load", async ({ page }) => {
   await page.addInitScript(() => {
-    window.localStorage.setItem("serverUrl", "http://mysite.com");
+    window.localStorage.setItem("serverUrl", "http://localhost:8005");
```
Comment (ypkang) [inline jaseci_studio/__e2e__/logs.spec.ts:10]: @Gim3l what do these URL do? Would localhost here work with tests and would it work with production?

**Review by ypkang — CHANGES_REQUESTED** [review]

File: .github/workflows/jaseci-studio-test.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Jaseci Studio Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```
Comment (ypkang) [inline .github/workflows/jaseci-studio-test.yml:9]: @Gim3l Could you update this to only trigger the tests if there are changes in the `jaseci_studio` folder? You can take a look at the jaseci_ai_kit github action workflow file for how to do that.

File: jaseci_studio/__e2e__/logs.spec.ts (hunk 1 above)
Comment (Gim3l) [reply jaseci_studio/__e2e__/logs.spec.ts:10]: Those are mock urls used for e2e testing. Yes, the localhost version works with the tests on github, the other link doesn't.

File: .github/workflows/jaseci-studio-test.yml (hunk 2 above)
Comment (Gim3l) [reply .github/workflows/jaseci-studio-test.yml:9]: Ok, I made the change.

**Review by ypkang — APPROVED** [review]

**Pull request merged**